incubator/appdaemon/0.1.0/test_values.yaml

```diff
@@ -16,7 +16,7 @@ podSecurityContext:
 
 env:
   URL: ""
-  Token: "" This is a Statping instance deployed as Helm chart
+  TOKEN: ""
```

Comment:
Why is TOKEN required here?

Reply:
I believe I'm setting docker variables here. Here is the docker config from the apps webpage.

https://appdaemon.readthedocs.io/en/latest/DOCKER_TUTORIAL.html

docker run --rm -it -p 5050:5050 \
  -e HA_URL="<your HA_URL value>" \
  -e TOKEN="<your TOKEN value>" \
  -e DASH_URL="http://$HOSTNAME:5050" \
  acockburn/appdaemon:latest

Reply:
Yes, you are setting environment variables (that is NOT a docker specific thing, it's a linux thing) here.

Try to investigate what is required and add only required env vars here.
you are now having only TOKEN  and not the rest, you should, as a maintainer, be able to explain why ...